server/boards/tests/routes/delete-board.test.ts

```diff
@@ -0,0 +1,84 @@
+import { BOBATAN_USER_ID, JERSEY_DEVIL_USER_ID } from "test/data/auth";
+import { CacheKeys, cache } from "server/cache";
+import { GORE_BOARD_ID, GORE_BOARD_METADATA } from "test/data/boards";
+import {
+  setLoggedInUser,
+  startTestServer,
+  wrapWithTransaction,
+} from "utils/test-utils";
+
+import debug from "debug";
+import request from "supertest";
+import router from "../../routes";
+
+const log = debug("bobaserver:test:boards:routes:delete-board-log");
+
+jest.mock("server/cache");
+jest.mock("handlers/auth");
+jest.mock("server/db-pool");
+jest.setTimeout(20000);
+
+describe("DELETE /board/:board_id", () => {
+  const server = startTestServer(router);
+
+  test("Should return 401 if user is not logged in", async () => {
+    const res = await request(server.app).delete(`/${GORE_BOARD_ID}`);
+
+    expect(res.status).toBe(401);
+  });
+
+  it("Should return 404 if board does not exist", async () => {
```

Comment:
Any reason you're switching between `test()` and `it()` here? It doesn't matter, but we should really pick one or the other and try to stay stylistically consistent for clarity.